src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -1156,7 +1163,7 @@ def create(
             schema_name = schema.name
 
         if name is None:
-            name = f"ucx_T{make_random(4)}".lower()
+            name = f"ucx_t{make_random(4).lower()}_{get_purge_suffix()}"
```

Comment:
done via properties. purge suffix is irrelevant here

Reply:
This was a bit of a head-scratcher, mostly because via the SQL interface doesn't allow these to be modified and I wasn't sure it was worth switching to the REST API.

That said, after poking around in the Watchdog a bit, I don't think functions are explicitly being scanned/nuked? Instead it looks like they're bycatch from the schema scanner?